src/routes/puzzles/[id]/play/+page.svelte

```
@@ -45,9 +47,14 @@
 			},
 			method: POST
 		});
+
+		if (!response.ok) {
+			return;
+		}
+
```

What is this change about? I think this silently hides errors which is unnecessary and makes debugging issues harder.

---

was in the process of adding toasts, but remembered you don't necessarily like them, so didn't continue

```suggestion
```